CME/dataobjects/CMEAccount.php

```diff
@@ -177,6 +178,34 @@ public function getAvailableCMECredits()
 	}
 
 	// }}}
+	// {{{ public function getCMEProgress()
+
+	public function getCMEProgress(RapCredit $credit)
+	{
+		require_once 'CME/dataobjects/CMEAccountCMEProgressWrapper.php';
+
+		$this->checkDB();
+
+		$sql = sprintf(
+			'select AccountCMEProgress.*
+			from AccountCMEProgress
+			where AccountCMEProgress.account = %s
+			and AccountCMEProgress.id in (
+				select progress from AccountCMEProgressCreditBinding
+				where AccountCMEProgressCreditBinding.credit = %s
+			)',
+			$this->db->quote($this->id, 'integer'),
+			$this->db->quote($credit->id, 'integer'),
+		);
+
+		return SwatDB::query(
+			$this->app->db,
+			$sql,
+			SwatDBClassMap::get('CMEAccountCMEProgressWrapper');
+		)->getFirst();
+	}
+
+	// }}}
```

Comment:
We discussed this in person. The answer is "no". We want to verify if credits belong to the same progress, or if no progress has started.